### ISSUE TITLE
Xliffmerge is not valid

### ISSUE THREAD

**Issue by danrald** [martinroob/ngx-i18nsupport#152]
This seems like an outstanding tool but it does not seem to work. I've done a sample project using the   documentation and I cannot seem to get it to work. If I issue 

> "npm run extrac-18n" 

which calls 

> "ng xi18n translatorNGXApp --i18n-format xlf --output-path i18n --i18n-locale en && xliffmerge --profile xliffmerge.json"

 I get an error

> ERROR: language "C:\development\TranslatorNGX\translatorNGXApp\node_modules\@ngx-i18nsupport\ngx-i18nsupport\xliffmerge\xliffmerge" is not valid

The xliffmerge file is there and just states "require('../src/xliffmerge/main.js')", I am not sure why it is failing. If I can't get this to work, looks like I will have to cancel using ngx-i18nsupport. Which would be too bad.

Cheers!
Dan

**Comment by martinroob** [comment]
Might be connected with #144 
Are you using yarn?
In general the usage of the command line tool is somewhat deprecated.
You should use the integrated ng builder.
What versions of angular and node are you using.

**Comment by MarcusRiemer** [comment]
Yes, this is a duplicate of #144. I am currently falling back to using this tool via CLI because the integrated ng builder seems to be [broken as of Angular 11](https://github.com/martinroob/ngx-i18nsupport/issues/167#issuecomment-726224125) :-(
